### DISCUSSION TITLE
MHI2_ER_AU57×_K3663_1

### DISCUSSION THREAD

**Discussion by thed3vil** [Mr-MIBonk/M.I.B._More-Incredible-Bash#296] [Q&A] [answered]
"I recently installed the MHI2_ER_AU57×_K3663_1 patch on my A6 C7, and everything appeared to be fine as the version information displayed 'train I MHI2_ER_AU57×_K3663_1.' However, after rebooting, CarPlay and Android Auto was not there as in smartphone interface . I attempted to access the engineering and green menus, but I couldn't find the FEC that was supposed to enable CarPlay and AA. What should I do to ensure that the FEC patch is applied correctly? Should I reflash the system, or is there a way to manually add the FEC?"

**Comment by harman-f** [comment]
You mean the AIO FW?
Install it again.

**Reply by thed3vil** [reply]
Thanks for your prompt reply , Audi MMI 4G CarPlay activation with MIB2 K3663 AIO update <- this is what i download , so do you mean , re-update again using red engineer menu ? 

1) can i edit my FEC for maps ? as i need ROW ? 23D00EE

2) this should be the extract files in my SD card ? or go into number 1 folder and copy to SD ?

![a6fw](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/25909253/8ee15c21-9714-4aa1-a7b1-c69c6c884db0)

**Reply by harman-f** [reply, marked as answer]
check addFec.txt
23D00EE should alread be included.

**Comment by alexandrualancea** [comment]
Где я могу скачать обновления програмного обезпечення для ауди а6 с7  миб2
